packages/tools/src/tools/segmentation/RectangleScissorsTool.ts

```diff
@@ -145,25 +148,44 @@ class RectangleScissorsTool extends BaseTool {
         },
       },
     };
-
     const viewportIdsToRender = getViewportIdsWithToolToRender(
       element,
       this.getToolName()
     );
 
-    this.editData = {
-      annotation,
-      segmentation,
-      segmentIndex,
-      segmentsLocked,
-      segmentColor,
-      segmentationId,
-      viewportIdsToRender,
-      handleIndex: 3,
-      movingTextBox: false,
-      newAnnotation: true,
-      hasMoved: false,
-    };
+    if (labelmapData.type === 'volume' || !labelmapData.type) {
+      const { volumeId } = representationData[
+        type
+      ] as LabelmapSegmentationDataVolume;
+      const segmentation = cache.getVolume(volumeId);
+      this.editData = {
+        annotation,
+        segmentation,
+        segmentIndex,
+        segmentsLocked,
+        segmentColor,
+        segmentationId,
+        viewportIdsToRender,
+        handleIndex: 3,
+        movingTextBox: false,
+        newAnnotation: true,
+        hasMoved: false,
+      };
+    } else {
+      this.editData = {
+        annotation,
+        currentImageId: viewport.getCurrentImageId(),
+        segmentIndex,
+        segmentsLocked,
+        segmentColor,
+        segmentationId,
+        viewportIdsToRender,
+        handleIndex: 3,
+        movingTextBox: false,
+        newAnnotation: true,
+        hasMoved: false,
+      };
+    }
```

Comment:
this is alsmot repeated code, this can better organized like 

```js
this.editData = {
  annotation,
  segmentIndex,
  segmentsLocked,
  segmentColor,
  segmentationId,
  viewportIdsToRender,
  handleIndex: 3,
  movingTextBox: false,
  newAnnotation: true,
  hasMoved: false,
};

// Conditionally 
if (labelmapData.type === 'volume' || !labelmapData.type) {
  const { volumeId } = representationData[type] as LabelmapSegmentationDataVolume;
  const segmentation = cache.getVolume(volumeId);
  
  this.editData.segmentation = segmentation;
} else {
  this.editData.currentImageId = viewport.getCurrentImageId();
}

```